### PR TITLE
Inertia legs

### PR DESCRIPTION
**Pull request by wuallenwu**
Adding legs with inertia and making sure they work in sim

### CODE REVIEW

**Comment by wuallenwu** [comment]
Additionally revised some mjcf stuff to make sure the automatic update is smoother

File: ksim/mjx_gym/envs/stompy_env/rewards.py [hunk 1]
```diff
@@ -108,6 +108,10 @@ def healthy_reward_fn(
     is_healthy = jp.where(state.q[2] > max_z, 0.0, is_healthy)
     healthy_reward = jp.array(params["weight"]) * is_healthy
 
+    # jax.debug.breakpoint()
```
Comment (budzianowski) [inline ksim/mjx_gym/envs/stompy_env/rewards.py]: remove

File: ksim/mjx_gym/envs/stompy_env/stompy.py [hunk 2]
```diff
@@ -83,6 +84,7 @@ def reset(self, rng: jp.ndarray) -> State:
         for key in self._reward_params.keys():
             metrics[key] = zero
 
+        # jax.debug.breakpoint()
```
Comment (budzianowski) [inline ksim/mjx_gym/envs/stompy_env/stompy.py]: remove

File: ksim/mjx_gym/experiments/stompy_walk.yaml [hunk 3]
```diff
@@ -21,8 +21,8 @@ reward_params:
     weight: 1.25
   rew_healthy:
     weight: 5.0
-    healthy_z_lower: 0.4
-    healthy_z_upper: 4.0
+    healthy_z_lower: -1.0
```
Comment (budzianowski) [inline ksim/mjx_gym/experiments/stompy_walk.yaml:24]: does this still apply?

File: ksim/mjx_gym/utils/rollouts.py [hunk 4]
```diff
@@ -123,6 +123,7 @@ def render_mujoco_rollout(
         act_rng, seed = jax.random.split(rng)
         obs = env._get_obs(mjx.put_data(model, data), ctrl)
         # TODO: implement methods in envs that avoid having to use mjx in a hacky way...
+        # print(obs)
```
Comment (budzianowski) [inline ksim/mjx_gym/utils/rollouts.py:126]: remove?

File: ksim/scripts/process_mjcf.py [hunk 5]
```diff
@@ -2,7 +2,7 @@
 """Defines common types and functions for exporting MJCF files.
 
 Run:
-    python ksim/scripts/process_mjcf.py /Users/allen/KScale/ksim/ksim/assets/newnew/robot_7dof_arm_merged_simplified.xml
+    python ksim/scripts/process_mjcf.py /Path/tp/
```
Comment (budzianowski) [inline ksim/scripts/process_mjcf.py]: fix?

File: ksim/scripts/process_mjcf.py [hunk 6]
```diff
@@ -34,28 +34,29 @@
 
 STOMPY_HEIGHT = 1.0
 
+# Comment out the joints that are not needed
 DEFAULT_STANDING = {
-    "torso roll": 2.58,
```
Comment (budzianowski) [inline ksim/scripts/process_mjcf.py:38]: Let's copy the sim setup?

File: ksim/scripts/process_mjcf.py [hunk 7]
```diff
@@ -71,59 +72,60 @@
...
-    },
-    "right wrist pitch": {
-        "lower": -1.5708,
-        "upper": 0.523599,
-    },
-    "right wrist yaw": {
-        "lower": -1.5708,
-        "upper": 0,
-    },
+    # "torso roll": {
```
Comment (budzianowski) [inline ksim/scripts/process_mjcf.py:77]: againt let's copy the sim setup?

File: ksim/scripts/process_mjcf.py [hunk 8]
```diff
@@ -250,23 +252,14 @@ def adapt_world(
...
-                mjcf.Joint(name="root_z", type="slide", axis=(0, 0, 1), limited=False).to_xml(),
-                mjcf.Joint(name="root_ball", type="ball", limited=False).to_xml(),
-            ]
-        )
+        # new_root_body = mjcf.Body(name="root", pos=(0, 0, STOMPY_HEIGHT), quat=(1, 0, 0, 0)).to_xml()
 
-        # Add imu site to the body - relative position to the body
-        # check at what stage we use this
-        new_root_body.append(mjcf.Site(name="imu", size=0.01, pos=(0, 0, 0)).to_xml())
+        # # Add imu site to the body - relative position to the body
```
Comment (budzianowski) [inline ksim/scripts/process_mjcf.py:255]: fix

File: ksim/scripts/process_mjcf.py [hunk 9]
```diff
@@ -250,23 +252,14 @@ def adapt_world(
...
+        # # check at what stage we use this
+        root.append(mjcf.Site(name="imu", size=0.01, pos=(0, 0, 0)).to_xml())
 
-        # Add the new root body to the worldbody
-        worldbody.append(new_root_body)
+        # # Add the new root body to the worldbody
```
Comment (budzianowski) [inline ksim/scripts/process_mjcf.py]: remove

File: ksim/scripts/process_mjcf.py [hunk 10]
```diff
@@ -321,24 +314,24 @@ def adapt_world(
...
-            [
-                ET.Element("framequat", name="orientation", objtype="site", noise="0.001", objname="imu"),
-                ET.Element("gyro", name="angular-velocity", site="imu", noise="0.005", cutoff="34.9"),
-                # ET.Element("framepos", name="position", objtype="site", noise="0.001", objname="imu"),
-                # ET.Element("velocimeter", name="linear-velocity", site="imu", noise="0.001", cutoff="30"),
-                # ET.Element("accelerometer", name="linear-acceleration", site="imu", noise="0.005", cutoff="157"),
-                # ET.Element("magnetometer", name="magnetometer", site="imu"),
-            ]
-        )
+        # sensors = root.find("sensor")
```
Comment (budzianowski) [inline ksim/scripts/process_mjcf.py:320]: add todo to add them later?

**Review by budzianowski — APPROVED** [review]
small nits but looking good

**Comment by wuallenwu** [comment]
@budzianowski fixed requested changes

**Pull request merged**